docs/source/hardware-interfaces.rst

```diff
@@ -0,0 +1,34 @@
+.. _hardware_interface_packages:
+
+Hardware Interface Packages
+===========================
+
+The Bluesky library does not provide direct support for communicating with real hardware.
+Instead, we define a high-level abstraction: the :ref:`Bluesky Hardware Interface <hardware_interface>`.
+This allows different experimentalists to use different hardware control systems.
+
+The following packages provide support for real hardware communication from Bluesky:
+
+=============  ========================================================================
+Instrbuilder_  Lightweight package with a focus on SCPI_.
+Ophyd_         Primary hardware interface package for Bluesky with a focus on EPICS_.
+Ophyd-Tango_   Tango_ integration for Bluesky.
```

Comment:
This is in such early stages that I think including alongside the others could be confusing. (I know I suggested including it.) It should either go in a separate table of experimental works-in-progress or have some disclaimer language like "incomplete and experimental early work".